ideascube/mediacenter/templates/mediacenter/document_detail.html

```diff
@@ -18,7 +18,7 @@
     {% elif document.preview %}
         <a href="{% media document 'original' %}"><img src="{% media document 'preview' %}" /></a>
     {% endif %}
-    <div class="text">{{ document.summary }}</div>
+    <div class="text">{{ document.summary|safe }}</div>
```

Comment:
But tinyMCE will send us valid html we don't want to escape. At first glance, I don't know how to make the distinction between valid html coming from tinyMCE and malicious one coming from evil user.

It's a real security problem, and we have to fix it. But I don't know how :/